From c02e76a2801d74f1e92ed0888163f660a5b5629f Mon Sep 17 00:00:00 2001
From: Haiqing Bai <Haiqing.Bai@windriver.com>
Date: Tue, 14 Jan 2020 07:19:51 +0000
Subject: [PATCH] filldir[64]: Replace __uaccess_begin with user_access_begin

In commit 3e62ebf57245(Avoid changing 'user_access_begin()' to
'user_access_begin(const void __user *ptr, size_t len)')
'__uaccess_begin' should be replaced with 'user_access_begin'

Fixes: 3e62ebf57245("Avoid changing 'user_access_begin()' to
'user_access_begin(const void __user *ptr, size_t len)'")

Signed-off-by: Haiqing Bai <Haiqing.Bai@windriver.com>
---
 fs/readdir.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fs/readdir.c b/fs/readdir.c
index 11a52e327e05..c172e0f1be7c 100644
--- a/fs/readdir.c
+++ b/fs/readdir.c
@@ -242,7 +242,7 @@ static int filldir(struct dir_context *ctx, const char *name, int namlen,
 	 */
 	if (unlikely(!access_ok(VERIFY_WRITE, dirent, sizeof(*dirent))))
 		goto efault;
-	__uaccess_begin();
+	user_access_begin();
 
 	if (dirent)
 		unsafe_put_user(offset, &dirent->d_off, efault_end);
@@ -331,7 +331,7 @@ static int filldir64(struct dir_context *ctx, const char *name, int namlen,
 	 */
 	if (unlikely(!access_ok(VERIFY_WRITE, dirent, sizeof(*dirent))))
 		goto efault;
-	__uaccess_begin();
+	user_access_begin();
 
 	if (dirent)
 		unsafe_put_user(offset, &dirent->d_off, efault_end);
-- 
2.17.1

